_locales/en/messages.json

```diff
@@ -758,6 +758,81 @@
     "configurationBatteryCapacityUnit": {
         "message": "Battery Capacity Unit"
     },
+    "configurationLaunch": {
+        "message": "Auto Launch Configuration"
+    },
+    "configurationLaunchVelocity": {
+        "message": "Velocity"
+    },
+    "configurationLaunchAccel": {
+        "message": "Acceleration"
+    },
+    "configurationLaunchMaxAngle": {
+        "message": "Max Angle"
+    },
+    "configurationLaunchDetectTime": {
+        "message": "Detect Time"
+    },
+    "configurationLaunchThr": {
+        "message": "Throttle"
+    },
+    "configurationLaunchIdleThr": {
+        "message": "Idle Throttle"
+    },
+    "configurationLaunchMotorDelay": {
+        "message": "Motor Delay"
+    },
+    "configurationLaunchSpinupTime": {
+        "message": "Spinup Time"
+    },
+    "configurationLaunchMinTime": {
+        "message": "Min Time"
+    },
+    "configurationLaunchTimeout": {
+        "message": "Time Out"
+    },
+    "configurationLaunchMaxAltitude": {
+        "message": "Max Altitude"
+    },
+     "configurationLaunchClimbAngle": {
+        "message": "Climb Angle"
```

Comment:
"Climb angle for launch sequence (degrees), is also restrained by global max_angle_inclination_pit" also see the other descriptions from here  [https://github.com/iNavFlight/inav/blob/56be63d5f1f220b4f650ec448f4e5d4d729e951f/src/main/fc/settings.yaml](https://github.com/iNavFlight/inav/blob/56be63d5f1f220b4f650ec448f4e5d4d729e951f/src/main/fc/settings.yaml)

Reply:
i dont get what you are trying to say. I already added that HELP description when you mouseover the help.
would you like to change that value also just like the others i add?